docs/connecting.txt

```diff
@@ -89,13 +89,15 @@ connection with the database superuser ``crate``.
     properties.put("ssl", "true");
     Connection conn = DriverManager.getConnection("crate://localhost:5432/", properties);
 
-In addition connection properties can be passed via the JDBC URL:
+In addition, connection parameters can be passed via the JDBC URL:
 
 .. code-block:: java
 
     Connection conn = DriverManager.getConnection("crate://localhost:5432/?property1=value1&property2=value2");
 
-CrateDB JDBC driver supports following properties:
+CrateDB JDBC driver supports all of `the PostgreSQL JDBC driver parameters`_,
+though not all of them will make sense for CrateDB. We have listed the most
+important ones here.
```

Comment:
where?

Reply:
this was just a partial fixup so I could switch branches :)